demos/storybook/stories/selection-toolbar/with-basic-usage.stories.ts

```diff
@@ -0,0 +1,30 @@
+import { text } from '@storybook/addon-knobs';
+
+export const withBasicUsage = (): any => ({
+    styles: [
+        `
+       ::ng-deep .pxb-selection-toolbar {
+           width: 600px !important;
+       }
+    `,
+    ],
+    template: `
+       <pxb-selection-toolbar [title]="title" [subtitle]="subtitle">
+        <ng-container pxb-menu>
+            <button mat-menu-item>
```

Comment:
Can you tighten up this example a bit to make it be less lines.  The following behaves the same and just makes the example a bit easier to read imo.

```
            <button mat-menu-item>Test Item 1</button>
            <button mat-menu-item>Test Item 2</button>
            <button mat-menu-item>Test Item 3</button>
```